sdk/core/azure-core-tracing-opentelemetry/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Release History
+## Version 1.0.0-beta.2 (2019-01-)
```

Comment:
Missing a day for this?

Reply:
I will have it updated before the release.